src/glwidget.h

```diff
@@ -113,6 +113,7 @@ public slots:
     void offsetChanged();
     void imageReady();
     void snapToGridChanged();
+    void toggleZoom(bool);
```

Comment:
I added this so that the scope can toggle the player zoom to "best fit". Maybe you can think of a better way?